src/Sylius/Bundle/AdminBundle/Resources/views/Country/_form.html.twig

```diff
@@ -1,5 +1,5 @@
+{{ form_errors(form) }}
 <div class="ui segment">
-    {{ form_errors(form) }}
```

Comment:
Why 🤔?

Reply:
Why not 🤔?
It contains all errors for the whole form, but gets rendered in a segment containing only `code` and `enabled` fields. Looked weird to I've moved it 🤷